similarity_api/src/sim/compute.py

```diff
@@ -101,17 +143,11 @@ def find_similar_goals(recipient_id, goal_name, alpha):
         FROM "Goals" g
         JOIN "Grants" gr ON g."grantId" = gr."id"
         JOIN "Recipients" r ON gr."recipientId" = r."id"
-        WHERE r."id" = :recipient_id AND g."name" IS NOT NULL;
+        WHERE r."id" = :recipient_id AND NULLIF(TRIM(g."name"), '') IS NOT NULL;
```

Comment:
can we have a newline before ON's and AND

Reply:
7913b81e9